inst/examples/datepicker/app.R

```diff
@@ -13,10 +13,11 @@ app <- shinyApp(
       ),
       f7DatePicker(
         inputId = "picker",
-        label = "Choose a date",
-        value = Sys.Date() - 7,
+        label = "Choose a date and time",
+        value = Sys.time() - 3600*24,
```

Comment:
@DivadNojnarg using things like `Sys.date()` and `Sys.time()` will cause issues with testing. Results will always be different, so reverting back to fixed dates (and times)